README.md

```diff
@@ -56,11 +56,59 @@ The structure should appear like so:
 ├── documentation
 └── readme.md
 ```
+## New Directory structure (effective July 10th 2017)
 
+Re-organization is being done to support upcoming profiles work and Data Plane swagger specs. The new structure will appear like below
```

Comment:
The directory structure does not show up correctly when you view it as .md. Text shows up in a single line. Could you please take a look?